src/main/java/nl/stil4m/mollie/domain/Subscription.java

```diff
@@ -0,0 +1,179 @@
+package nl.stil4m.mollie.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Date;
+
+/**
+ * @see https://www.mollie.com/en/docs/reference/subscriptions/get
+ * @author stevensnoeijen
+ */
```

Comment:
Please remove the author. We have Git for this.

Reply:
removed @author from code